iceoryx_posh/include/iceoryx_posh/runtime/posh_runtime.hpp

```diff
@@ -62,14 +62,24 @@ class PoshRuntime
     /// @return name of the reistered application
     std::string getInstanceName() const noexcept;
 
+    /// @deprecated #240
     /// @brief find all services that match the provided service description
     /// @param[in] serviceDescription service to search for
     /// @param[out] instanceContainer container that is filled with all matching instances
     /// @return cxx::expected<Error> Error, if any, encountered during the operation
     /// Error::kPOSH__SERVICE_DISCOVERY_INSTANCE_CONTAINER_OVERFLOW : Number of instances can't fit in instanceContainer
     /// Error::kMQ_INTERFACE__REG_UNABLE_TO_WRITE_TO_ROUDI_MQ : Find Service Request could not be sent to RouDi
-    cxx::expected<Error> findService(const capro::ServiceDescription& serviceDescription,
-                                     InstanceContainer& instanceContainer) noexcept;
+    [[gnu::deprecated]] cxx::expected<Error> findService(const capro::ServiceDescription& serviceDescription,
```

Comment:
@nihalchari You can remove this deprecated function because we will not need it for the upcoming 1.0 release

Reply:
okay, i will remove it